shibboleth-idp/Dockerfile

```diff
@@ -19,7 +19,8 @@ ENV JETTY_HOME=/opt/jetty-home \
   JETTY_BASE=/opt/shib-jetty-base \
   PATH=$PATH:$JRE_HOME/bin
 
-RUN yum -y update \
+RUN yum-config-manager --disable updates \
+  && yum -y update \
```

Comment:
I'm not too familiar with centos. Does `--disable updates` disable `yum -y update` ??

Reply:
It disables `updates` repo where bugfixes and smaller updates to the centos packages are located. It will still do the update on packages but only from base repository (and some others) where there are less changes to the packages. This is to make it a bit more stable. It was part of Viacheslav's code so I suppose he had some issues with updates on some packages.